uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -184,6 +184,16 @@ define([
       };
 
       /**
+       * Checks if user can upload more file, it totals the number of already
+       * uploaded files and those which are in queue and compares it to limit.
+       *
+       * @return {Boolean} true is user can upload more else false
+       */
+      this.canUploadMore = function () {
```

Comment:
all good, but please keep the methods in alphabetical order, first `this.canSubmit, then `this.canUploadMore`